views/settings/advanced.html

```diff
@@ -159,6 +159,59 @@
         </p>
       </td>
     </tr>
+    <!-- inactive subscribers -->
+    <tr>
+      <th scope="row">
+        <label>
+          <%= __('Stop sending to inactive subscribers') %>
+        </label>
+        <p class="description">
+          <%= __("Gmail, Yahoo and other email providers will treat your emails like spam if your subscribers don't open your emails in the long run. This option will mark your subscribers as Inactive and MailPoet will stop sending to them.") %>
+        </p>
+      </th>
+      <td>
+        <p>
+          <label>
+            <input
+              type="radio"
+              name="inactive_subscriber_days"
+              value=""
+              <% if not(settings.inactive_subscriber_days) %>checked="checked"<% endif %>
```

Comment:
I think that `inactive_subscribers_days` does not describe exactly what the settings does. Maybe we could use something like:
`DEACTIVATE_SUBSCRIBER_AFTER_INACTIVE_DAYS`  What do you think?